examples/variables.tfvars

```diff
@@ -0,0 +1,15 @@
+credentials_path = "/Users/ocervello/rnm-shared-devops-fd8a7717d6e8.json"
```

Comment:
Don't commit your tfvars. If you want to include an example, you should redact all the values and call it `example.tfvars`.

Reply:
Oops, ok.